tool/jt.rb

```diff
@@ -1208,12 +1209,12 @@ def test_bundle(*args)
             # add bin from gem_home to PATH
             'PATH'     => [File.join(gem_home, 'bin'), ENV['PATH']].join(File::PATH_SEPARATOR))
 
-          run_ruby(environment, '-Xexceptions.print_java=true',
-              '-S', 'gem', 'install', '--no-document', 'bundler', '-v', '1.14.6', '--backtrace')
-          run_ruby(environment, '-Xexceptions.print_java=true',
-              '-J-Xmx512M','-S', 'bundle', 'install')
-          run_ruby(environment, '-Xexceptions.print_java=true',
-              '-S', 'bundle', 'exec', 'rake')
+          run_ruby(environment, '-Xexceptions.print_java=true', *('--jdebug' if jdebug),
+                   '-S', 'gem', 'install', '--no-document', 'bundler', '-v', '1.14.6', '--backtrace')
+          run_ruby(environment, '-Xexceptions.print_java=true', *('--jdebug' if jdebug), '-J-Xmx512M',
```

Comment:
I would normally pass VM options (`-J`) before Ruby options (`-X`).
Could you move the `-J-Xmx512M` in front?
Also, why is the heap limit needed?

Reply:
done